docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Set_an_Observation.md

```diff
@@ -0,0 +1,150 @@
+# Set an Observation
+
+This section describes how to set an Observation in your Azure IoT Central application.
+
+Setting an Observation lets your devices know what value changes and for what particular resources they need to notify you about. An Observation can be set for any or all the components of the [data model](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings/): objects, objects instances, and resources. Whenever there is a change in values, a device will send a Notify message to Coiote DM, which in turn will transfer it to Azure IoT Central.
+
+In this section, you learn how to:
+* Assign a device template that acts as a translator between Coiote DM and Azure IoT Central
+* Create a device group, because an Observation is set at the group level
+* Run Jobs to add, edit or delete Observations
+* See value changes for the observed resources
+
+## Prerequisites
+
+1. An active Azure IoT Central with hub owner access permissions.
+2. A Coiote DM user account with permissions to use the integration extension.
+3. A device group created in Coiote DM.
+4. [A configured extension between Coiote DM and Azure Iot Central](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Configuring_Azure_IoT_Central_integration_extension/).
+5. [A configured integration template in Coiote DM](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates/).
+5. [A connected device](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Azure_IoT_Central_integration/Device_operations/Overview/).
+
+## Assign a device template
+
+To enable correct communication between Coiote DM and Azure IoT Central, you need two templates. The first template configures Coiote DM integration with Azure. (Follow [this instruction](https://iotdevzone.avsystem.com/docs/Azure_IoT_Integration_Guide/Configure_integration_templates/Azure_integration_templates/) to learn how to configure it.) The second template builds on the first one and allows to provide information that is used only in Azure IoT Central, e.g., that a given resource is temperature and is measured in Celsius.
+
+### Download LwM2M template in Coiote DM (UPDATE!!!)
```

Comment:
It seems this has changed. Now you can download the template in Hyperscaler I. C., right?